website/common/locales/ach/groups.json

```diff
@@ -14,6 +14,7 @@
     "askQuestionGuild": "Ask a Question (Habitica Help guild)",
     "contributing": "Contributing",
     "faq": "FAQ",
+    "features": "Features",
```

Comment:
@jbusa22 adding the string is the correct thing to do but it should be added only to the `locales/en/groups.json` file